### PR TITLE
Release preparation

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: examples/eVolve/rawVolModelRenderer.cpp [hunk 1]
```diff
@@ -66,7 +66,7 @@ static void renderSlices( const SliceClipper& sliceClipper )
 
 
 void RawVolumeModelRenderer::_putVolumeDataToShader(
-        const VolumeInfo&   volumeInfo,
+    const VolumeInfo&   volumeInfo,
         const float         sliceDistance,
```
Comment (tribal-tec) [inline examples/eVolve/rawVolModelRenderer.cpp]: indentation?

File: examples/triply/typedefs.h [hunk 2]
```diff
@@ -40,14 +40,12 @@
...
 #  define PLYLIBASSERT  LBASSERT
 #  define PLYLIBERROR   LBERROR
 #  define PLYLIBWARN    LBWARN
 #  define PLYLIBINFO    LBINFO
 #else
 #  include <GL/glew.h>
-#  include <vmmlib/vmmlib.hpp>
 #  ifdef _WIN32
 #    include <Winsock2.h>
 #    include <Windows.h>
```
Comment (tribal-tec) [inline examples/triply/typedefs.h]: those two headers also don't belong here imo

Comment (eile) [inline examples/triply/typedefs.h]: Can you (re)move them in another commit, verifying the Windows build still works. They've been here for quite a bit.

File: examples/eVolve/rawVolModelRenderer.cpp (hunk 1 above)
Comment (eile) [inline examples/eVolve/rawVolModelRenderer.cpp]: done

File: examples/triply/typedefs.h (hunk 2 above)
Comment (tribal-tec) [inline examples/triply/typedefs.h]: Will do.

**Comment by tribal-tec** [comment]
+1

**Comment by eile** [comment]
Retest this please.

File: examples/triply/typedefs.h (hunk 2 above)
Comment (tribal-tec) [inline examples/triply/typedefs.h]: Can be removed

**Comment by eile** [comment]
Retest this please.

**Comment by eile** [comment]
Retest this please.
